server/src/test/java/org/eclipse/lsp/cobol/service/delegates/completions/CopybookNameCompletionTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.service.delegates.completions;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import org.eclipse.lsp.cobol.service.CobolDocumentModel;
+import org.eclipse.lsp.cobol.service.CopybookNameService;
+import org.eclipse.lsp.cobol.service.CopybookNameServiceImpl;
+import org.eclipse.lsp.cobol.service.delegates.validations.AnalysisResult;
+import org.eclipse.lsp4j.*;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/** Test to check SubroutineCompletion */
+class CopybookNameCompletionTest {
+  private static final String TEXT = "       COPYBOOK  \n";
+
+  @Test
+  void testCopybookNameCompletion() {
+    CopybookNameService copybookNameService = new CopybookNameServiceImpl(null, null, null);
+
+    copybookNameService.setNames(ImmutableList.of("Copy1", "copy2", "no-copy"));
```

Comment:
I see now.
Here you testing completion business logic and it doesn't matter what's inside `CopybookNameService`, so you can just mock the service.